### PR TITLE
Add QR code from qrcode npm package

### PR DESCRIPTION
**Pull request by ssomalraju2**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -19,6 +19,6 @@
     "src/**/*.tsx",
     "src/**/*.vue",
     "src/*.d.ts"
-  ],
+, "src/scanQRCode.js", "src/scanQRCode.js"  ],
```
Comment (scvnc) [inline tsconfig.json:22]: Lets turn scanQRCode into a typescript file so that we don't have to define it in this file :)

File: index.html [hunk 2]
```diff
@@ -9,5 +9,6 @@
   <body>
     <div id="app"></div>
     <script type="module" src="/src/main.ts"></script>
+    <script type="application/javascript" src="scanQRCode.js"></script>
```
Comment (scvnc) [inline index.html:12]: you should be able to remove this script tag.  The vue file will be able to reference

File: src/scanQRCode.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+var QRCode = require('qrcode')
+var canvas = document.getElementById('canvas')
+
+QRCode.toCanvas(canvas, 'sample text', function (error: any) {
```
Comment (scvnc) [inline src/scanQRCode.ts:4]: we need to figure out a way to make this execute only when we know the <canvas> is on the DOM.

**Review by scvnc — CHANGES_REQUESTED** [review]

File: yarn.lock [hunk 4]
```diff
@@ -78,787 +73,865 @@
...
+"is-core-module@^2.9.0":
+  "integrity" "sha512-Erxj2n/LDAZ7H8WNJXd9tw38GYM3dv8rk8Zcs+jJuxYTW7sozH+SS8NtrSjVL1/vpLvWi1hxy96IzjJ3EHTJJg=="
+  "resolved" "https://registry.npmjs.org/is-core-module/-/is-core-module-2.10.0.tgz"
+  "version" "2.10.0"
+  dependencies:
+    "has" "^1.0.3"
+
+"is-fullwidth-code-point@^3.0.0":
+  "integrity" "sha1-8Rb4Bk/pCz94RKOJl8C3UFEmnx0="
+  "resolved" "https://pkgs.dev.azure.com/osh-inc/_packaging/osh-inc/npm/registry/is-fullwidth-code-point/-/is-fullwidth-code-point-3.0.0.tgz"
```
Comment (scvnc) [inline yarn.lock:490]: we also need to remove this because this is an internal npm feed.

Comment (scvnc) [reply yarn.lock:490]: I will pair with you on this tomorrow

File: tsconfig.json (hunk 1 above)
Comment (geppettodivacin) [reply tsconfig.json:22]: This can be removed now, right?

**Review by scvnc — APPROVED** [review]

**Pull request merged**